clang/test/dpct/query_api_mapping/Math/test-after11.6.cu

```diff
@@ -0,0 +1,21 @@
+// UNSUPPORTED: cuda-8.0, cuda-9.0, cuda-9.1, cuda-9.2, cuda-10.0, cuda-10.1, cuda-10.2, cuda-11.0, cuda-11.1, cuda-11.2, cuda-11.3, cuda-11.4, cuda-11.5
+// UNSUPPORTED: v8.0, v9.0, v9.1, v9.2, v10.0, v10.1, v10.2, v11.0, v11.1, v11.2, v11.3, v11.4, v11.5
+
+// RUN: dpct --cuda-include-path="%cuda-path/include" --query-api-mapping=__hfma | FileCheck %s -check-prefix=HFMA
+// HFMA: CUDA API:
+// HFMA-NEXT:   __hfma(h1 /*__half*/, h2 /*__half*/, h3 /*__half*/);
+// HFMA-NEXT:   __hfma(b1 /*__nv_bfloat16*/, b2 /*__nv_bfloat16*/, b3 /*__nv_bfloat16*/);
+// HFMA-NEXT: Is migrated to (with some neccessary option):
+// HFMA-NEXT:   sycl::ext::intel::math::hfma(h1, h2);
+// HFMA-NEXT:   b1 * b2 + b3;
+
+// RUN: dpct --cuda-include-path="%cuda-path/include" --query-api-mapping=__hfma_sat | FileCheck %s -check-prefix=HFMA_SAT
+// HFMA_SAT: CUDA API:
+// HFMA_SAT-NEXT:   __hfma_sat(h1 /*__half*/, h2 /*__half*/, h3 /*__half*/);
+// HFMA_SAT-NEXT:   __hfma_sat(b1 /*__nv_bfloat16*/, b2 /*__nv_bfloat16*/, b3 /*__nv_bfloat16*/);
+// HFMA_SAT-NEXT: Is migrated to (with some neccessary option):
```

Comment:
user will be mislead by  " some necessary option",  how about list the options explicitly, (the option can be embed in test cases in example/ folder)

Reply:
done